matchbox_socket/src/webrtc_socket/native/message_loop.rs

```diff
@@ -83,6 +85,9 @@ async fn message_loop_impl(id: PeerId, config: &WebRtcSocketConfig, channels: Me
                 if let Some(event) = message {
                     debug!("{:?}", event);
                     match event {
+                        PeerEvent::IdAssigned(peer_uuid) => {
+                            id_tx.send(peer_uuid.to_owned()).await.unwrap();
```

Comment:
I think a `try_send` would probably be better here. If the channel is full (which it shouldn't be as long as the server is behaving), we don't want the loop to stop here.